src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -14,6 +16,7 @@
 @Table (name = "shopping_carts")
 public class ShoppingCart {
     @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
```

Comment:
I think, you shouldn't change this field. If without that you have exception, try change persist() in ShoppingCartDaoImpl to save()